### PR TITLE
Add simple example content

### PR DESCRIPTION
**Pull request by whmii**
screencapture-localhost-4567-examples-1471617284352

### CODE REVIEW

**Comment by ColeTownsend** [comment]
visual crit here, the quote + blue glow feel out of sync with the emoji/stark style to me, that's my gut reaction. Maybe the OSX default screenshot shadow would be more fitting w/ the emojis as well?

File: source/assets/stylesheets/base/_typography.scss [hunk 1]
```diff
@@ -71,3 +71,7 @@ hr {
   border-top: 0;
   margin: $base-spacing 0;
 }
+
+blockquote {
+  font-style: italic;
```
Comment (ColeTownsend) [inline source/assets/stylesheets/base/_typography.scss]: might be worth also adjusting the line-height for this

File: source/assets/stylesheets/components/_example.scss [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  margin-bottom: $small-spacing;
+  text-align: center;
+}
+
+.example__sub-heading {
+  font-family: $base-font-family;
+  font-size: modular-scale(-2);
+  font-weight: 300;
+  font-style: italic;
+  margin-bottom: $small-spacing;
```
Comment (ColeTownsend) [inline source/assets/stylesheets/components/_example.scss]: double margin

File: source/assets/stylesheets/components/_hero.scss [hunk 3]
```diff
@@ -13,12 +13,16 @@
...
 }
 
-.hero__flex-container {
+.hero__flex-container,
+.hero__flex-container--small {
   display: flex;
   flex-direction: column;
   justify-content: space-between;
-  min-height: calc(90vh - #{$aproximate-nav-height});
   padding: 10vh 0;
```
Comment (ColeTownsend) [inline source/assets/stylesheets/components/_hero.scss]: @include pading 10vh null?

File: source/assets/stylesheets/components/_hero.scss [hunk 4]
```diff
@@ -13,12 +13,16 @@
...
+}
+
+.hero__flex-container {
+  min-height: calc(90vh - #{$aproximate-nav-height});
 
   @include grid-media($neat-grid-tablet) {
     padding: 12vh 0;
```
Comment (ColeTownsend) [inline source/assets/stylesheets/components/_hero.scss]: same as above

**Comment by whmii** [comment]
@ColeTownsend good to know. I want to work in some of color shadows in to this brand but using it here may be too heavy handed. I actually started w/ OS X style shadows based on http://mcmahan.me/css-sass-styling-for-code-blocks

**Comment by whmii** [comment]
Also @ColeTownsend do you think a page like this is useful/helpful? I'm not sure this thing should even exist.

**Comment by ColeTownsend** [comment]
@whmii just seeing these 0.o I like the examples offered, some people want handholding

**Comment by whmii** [comment]
cool

**Pull request merged**